author_postprocessing/author_postprocessing.py

```diff
@@ -0,0 +1,217 @@
+# coding=utf-8
+# This file is part of codeface-extraction, which is free software: you
+# can redistribute it and/or modify it under the terms of the GNU General
+# Public License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2015-2017 by Claus Hunsen <hunsen@fim.uni-passau.de>
+# Copyright 2020 by Thomas Bock <bockthom@cs.uni-saarland.de>
+# All Rights Reserved.
+"""
+This file is able to disambiguate authors after the extraction from the Codeface database was performed. A manually created disambiguation file is used to disambiguate the authors in all the extracted files of a project.
```

Comment:
I don't know whether this is a GitHub problem or if this line is just really long. If that is the case, I think you should split the line for a more readable intro.

Reply:
This line is really too long. I've just forgotten to add line breaks :wink: 
Thanks for spotting this @hechtlC !

I have already fixed this. In addition, I have added a more detailed description describing the format of the disambiguation file, including a small example.

Reply:
I just had a look at the new changes and didn't find any issues with it. So in my opinion this is ready for merging if @clhunsen approves.
Thanks again @bockthom!